### PR TITLE
Allow updating requistion details in the UpdateComputationDetails call.

### PR DESCRIPTION
**Pull request by wangyaopw**
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/66)

### CODE REVIEW

**Comment by wangyaopw** [comment]
This is a followup RP of https://github.com/world-federation-of-advertisers/cross-media-measurement/pull/34

File: src/main/proto/wfa/measurement/internal/duchy/computations_service.proto [hunk 1]
```diff
@@ -115,6 +116,16 @@ message UpdateComputationDetailsRequest {
...
+  // Updated requisition details.
+  message RequisitionDetailUpdate {
+    // External ID of the data provider.
+    string external_data_provider_id = 1;
+    // External ID of the requisition.
+    string external_requisition_id = 2;
+    // New value of the requisition details.
+    RequisitionDetails details = 3;
+  }
+  repeated RequisitionDetailUpdate requisition_detail_update = 3;
```
Comment (zachcwc) [inline src/main/proto/wfa/measurement/internal/duchy/computations_service.proto:128]: Why is this repeated? Are Requisition detail updates batched?

Comment (wangyaopw) [reply src/main/proto/wfa/measurement/internal/duchy/computations_service.proto:128]: A computation has multiple requisitions, each requisition has its own detail to update. We update them together in a single rpc call.

Comment (zachcwc) [reply src/main/proto/wfa/measurement/internal/duchy/computations_service.proto:128]: Makes sense, thanks

**Review by zachcwc — APPROVED** [review]

**Review by kungfucraig — APPROVED** [review]

**Pull request merged**